frontend/javascripts/components/adapt_viewport_metatag.ts

```diff
@@ -1,13 +0,0 @@
-import { useEffect } from "react";
```

Comment:
see https://github.com/scalableminds/webknossos/pull/4040 for context. I think the removal of the meta tag is not necessary anymore. I think, the original idea was so that one can zoom in and pan around WK with the "desktop dimensions", but the new mobile button bar is the better solution probably.